pkgs/development/compilers/adoptopenjdk-bin/jdk-linux-base.nix

```diff
@@ -74,10 +47,7 @@ let result = stdenv.mkDerivation rec {
 
     # Remove embedded freetype to avoid problems like
     # https://github.com/NixOS/nixpkgs/issues/57733
-    rm $out/lib/libfreetype.so
-
-    # for backward compatibility
-    ln -s $out $out/jre
```

Comment:
Do we not need this backwards compatibility any more?

Reply:
I couldn't figure out what the backwards compatibility concern was here, since `openjdk11` does not have this symlink. Also, users are unlikely to use `adoptopenjdk` directly anymore, so I felt that backwards compatibility was less important.

Reply:
Maybe @taku0 who added the line initially has an opinion?

Reply:
Oracle JDK and other JDKs had `jre` folder and some packages depend on it.  I forgot which packages depend on it.

Reply:
If `openjdk11` doesn't have it, and no one's complaining, then I don't think `adoptopenjdk` needs it either.